README.md

```diff
@@ -0,0 +1,5 @@
+# Questions about `pull-request`
+ 
+- Is there any example (repository, organization, ...) that shows the use of pull-requests in a classroom?
+- Do you think it would be easier to explain people with no technical background the use of git workflow using an IDE (like Fork) more than using terminal commands?
+- I'd like to use github classroom to organize all kinds of exercises for all types of audiences (non-technical included). What do you think (in your experience) that would be a better (easier to understand and apply for them)approach for this? 
```

Comment:
What kind of assignments would you distribute? Depending on your department, Google Classroom might be a better fit for distributing non-technical assignments.